webrecorder/browsertrix
# Add QA columns to archived item list

- Resolves https://github.com/webrecorder/browsertrix-cloud/issues/1672
- Fixes https://github.com/webrecorder/browsertrix-cloud/issues/1679



### Changes
- Shows QA review status, latest QA state, and QA run count in archived item list. "Created by" column replaced for space and moved to "Date Created" tooltip
- Enables sort archived items by QA columns
- Adds tooltip to all columns except name
- Hides non-applicable columns in when archived items are filtered to uploads only
- Fixes issue where tooltips weren't activated on hover when using `btrix-table`
- Minor refactor to reuse status labels/colors, renames page review -> page approval to clarify

### Manual testing

1. Log in as any role
2. Go to "Archived Items". Verify "QA Rating" and "Analysis Runs" columns are shown
3. Hover over "QA Rating" and "Analysis Runs" values. Verify tooltip is shown with rating scale and latest analysis run state, respectively
4. Hover over created date. Verify "created by" name is shown
5. Click archived item. Verify navigation to detail page as expected

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Archived Items - All  | <img width="1305" alt="Screenshot 2024-04-22 at 10 36 25 AM" src="https://github.com/webrecorder/browsertrix/assets/4672952/4b41c919-3b86-43d7-bb7b-0de6eaff6aab"> |
| Archived Items - All | <img width="321" alt="Screenshot 2024-04-17 at 3 03 18 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/7b8e07fa-72d0-476f-9eaf-3934d0442a99"> |
